refactor: :recycle: add book parts "Workshop overview" and "Pre-workshop tasks" (#122)
rostools/github-intro: pre-workshop/git-and-github.qmd

> 
> # Introduction to version control, Git, and GitHub {#sec-git-and-github}

Reviewer: ```suggestion
# Version control with Git and GitHub {#sec-git-and-github}
```

It's a bit too long, doesn't look good when rendered.
